pulp_ansible/app/tasks/collections.py

```diff
@@ -711,8 +711,8 @@ async def _find_all_collections(self):
 
     async def _should_we_sync(self):
         """Check last synced metadata time."""
-        msg = _("no-op: Checking if remote changed since last sync.")
-        noop = ProgressReport(message=msg, code="noop")
+        msg = _("no_change: Checking if remote changed since last sync.")
+        noop = ProgressReport(message=msg, code="sync.no_change")
```

Comment:
I didn't expect this, but I'm ok with it.

Reply:
As an explanation, since more than one logical "task" could be going on within a literal "task", we've been updating progress reports to be specific about what progress they are reporting.

A publish can now happen in the same task as a sync, so therefore we want to make the progress reporting conventions more specific.

But you're right, that isn't clear from just the PR description and the issue.  Apologies, this is wrapping up a few different things at once.

Reply:
It's no problem. Thank you @dralley !